firefox-ios/firefox-ios-tests/Tests/XCUITests/DatabaseFixtureTest.swift

```diff
@@ -23,18 +23,7 @@ class DatabaseFixtureTest: BaseTestCase {
         super.setUp()
     }
 
-    /* Disabled due to issue with db: 8281*/
-    /*func testOneBookmark() {
```

Comment:
Will we enable this test in the future?

Reply:
That particular scenario is covered in our bookmarks tests.

Reply:
Can we remove it then? here or in a follow up PR...

Reply:
@dragosb01 Could you please ensure that this deleted test isn't referred in TestRail?

Reply:
This test has no reference in TR